app/models/government.rb

```diff
@@ -1,8 +1,12 @@
 class Government < ActiveRecord::Base
+  DISTANT_FUTURE = Date.today + 100.years
```

Comment:
While in this instance it won't be a problem, just want to add a note that we have run into issues with saving relative dates into constants is that if the application isn't restarted the date will never be recalculated. The chance of this application not being restarted in 100 years is minimal though.